### PR TITLE
Update Android Auto docs to mention AAOS and mention minimum version for Wear OS

### PR DESCRIPTION
**Pull request by dshokouhi**
* Mention the minimum supported Wear OS version
* Mention Android Automotive OS (AAOS) and distinguish between the 2 as necessary
* Move favorites up as it is part of the home screen
* Briefly mention supported sensors
* Add notes for AAOS specifics

### CODE REVIEW

File: docs/android-auto/android-auto.md [hunk 1]
```diff
@@ -30,11 +30,21 @@ In order to use this integration you will need a phone as well as a vehicle with
 The amount of entities shown will depend on the imposed limit set by the vehicle.
 :::
 
+### Favorites
+
+If you would like to gain quick access to certain entities you can select some entities to be shown in the app. Simply go to [Settings](https://my.home-assistant.io/redirect/config/) > Companion App > Android Auto Favorites (or Driving Favorites when parked) and select the entities you wish to view. Once you have added favorites the next time you start the Home Assistant app in the vehicle you will be shown your favorite entities. From there you can toggle entities, navigate to an entity, view all entities and also change your server.
```
Comment (jpelgrom) [inline docs/android-auto/android-auto.md:35]: >\> Companion App > Android Auto Favorites (or Driving Favorites when parked)

- This should use sentence case now (Companion app, Android Auto favorites)
- "when parked" seems incorrect, as this title is always used when using Automotive?

File: docs/android-auto/android-auto.md [hunk 2]
```diff
@@ -30,11 +30,21 @@ In order to use this integration you will need a phone as well as a vehicle with
 The amount of entities shown will depend on the imposed limit set by the vehicle.
 :::
 
+### Favorites
+
+If you would like to gain quick access to certain entities you can select some entities to be shown in the app. Simply go to [Settings](https://my.home-assistant.io/redirect/config/) > Companion App > Android Auto Favorites (or Driving Favorites when parked) and select the entities you wish to view. Once you have added favorites the next time you start the Home Assistant app in the vehicle you will be shown your favorite entities. From there you can toggle entities, navigate to an entity, view all entities and also change your server.
+
+`binary_sensor` and `sensor` entities can be added to favorites in order to view their state in the driving interface.
```
Comment (jpelgrom) [inline docs/android-auto/android-auto.md:37]: Not wrong but maybe also mention that other entities (from the supported list above) can be selected as favorites as well?

File: docs/android-auto/android-auto.md [hunk 3]
```diff
@@ -52,8 +62,9 @@ Notifications in Android Auto share settings with your phone and do not support
             importance: high
 ```
 
-### Favorites
+### Sensors
 
-If you would like to gain quick access to certain entities you can select some entities to be shown in the app. Simply go to [Settings](https://my.home-assistant.io/redirect/config/) > Companion App > Android Auto Favorites and select the entities you wish to view. Once you have added favorites the next time you start the Home Assistant app in the car you will be shown your favorite entities. From there you can toggle entities, navigate to an entity, view all entities and also change your server.
+The sensors offered for AA and AAOS are described on the main sensor page. The following list of sensors are currently supported.
```
Comment (jpelgrom) [inline docs/android-auto/android-auto.md:67]: > The following list of sensors are currently supported.

This suggests to me that the main sensor page describes all sensors, _of which_ only the following two are supported. That's incorrect, they're more specific car sensors.

**Review by jpelgrom — COMMENTED** [review]
Somehow missed this, sorry for the delay.

Maybe it is also a good idea to rename the sidebar entry to something like "Android Auto/Automotive"?

File: docs/android-auto/android-auto.md (hunk 1 above)
Comment (dshokouhi) [reply docs/android-auto/android-auto.md:35]: > * "when parked" seems incorrect, as this title is always used when using Automotive?

actually more a safety precaution but also AAOS users cannot access favorites when the car isn't parked

Comment (jpelgrom) [reply docs/android-auto/android-auto.md:35]: True, but "when parked" also suggests that the title changes if you're using Android Auto and are parked.

File: docs/android-auto/android-auto.md (hunk 3 above)
Comment (dshokouhi) [reply docs/android-auto/android-auto.md]: updated to make it more clear

File: docs/android-auto/android-auto.md (hunk 2 above)
Comment (dshokouhi) [reply docs/android-auto/android-auto.md]: ok mentioned they can be added

File: docs/android-auto/android-auto.md (hunk 1 above)
Comment (dshokouhi) [reply docs/android-auto/android-auto.md]: rearranged the text so it should be more clear now :)

**Comment by dshokouhi** [comment]
> Maybe it is also a good idea to rename the sidebar entry to something like "Android Auto/Automotive"?

Done, I also did a slight rearrangement of the order of items as it seemed odd keeping automotive near the bottom visually. Also ok to rearrange it again but I think we should maintain the "core" order above.

**Comment by jpelgrom** [comment]
Re-ordering the menu, now you've made it difficult for yourself 😄

I can understand moving the vehicle interface up, but such a large distance between Apple Watch and Wear OS also looks weird in my opinion. Suggestion: core > Apple Watch > Wear OS > (CarPlay, if it is merged) > Android Auto/Automotive > Meta Quest > Troubleshooting?

**Comment by dshokouhi** [comment]
> Re-ordering the menu, now you've made it difficult for yourself 😄

😂 



> I can understand moving the vehicle interface up, but such a large distance between Apple Watch and Wear OS also looks weird in my opinion. Suggestion: core > Apple Watch > Wear OS > (CarPlay, if it is merged) > Android Auto/Automotive > Meta Quest > Troubleshooting?

I agree thats a lot better :)

**Review by jpelgrom — APPROVED** [review]

**Pull request merged**